star_profile.cpp

```diff
@@ -127,20 +151,20 @@ void ProfileWindow::OnPaint(wxPaintEvent& WXUNUSED(evt))
 
     int i;
     int *profptr;
-    wxString label;
+    string profileLabel;
     switch (this->mode) {  // Figure which profile to use
     case 0: // mid-row
     default:
         profptr = midrow_profile;
-        label = _("Mid row");
+        profileLabel = "Mid row";

```

Comment:
_("Mid row") is important since this marks the string for translation (gettext).  We need to keep the _() !
_() is a macro that expands to wxGetTranslation() so if the user has selected a different language, "Mid row" is replaced by the corresponding localized version of the text.